x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/utils/api_token_sort.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IApiToken } from '../types';
+
+export const apiTokenSort = (apiTokenA: IApiToken, apiTokenB: IApiToken): number => {
```

Comment:
Do we want to write a test file for this? I don't feel strongly enough to make it a blocker, but it might be nice to have just to get a "human readable" example of how the sort function works. Sort functions mystify me even when I've written them lmao :| Very similar to my relationship with regexes

Reply:
Yeah, I actually noticed it was missing a test and wondered the same thing. Since it's indirectly tested as part of the credentials list and had 100% code coverage I ended up leaving it as is. I think it'd be worth adding a test though. The fact that you wondered the same thing is a clear indication that we should have it.

Reply:
c52c139

Reply:
Awesome - the end assertion really helped me visualize order!